tests/config_template/copier.yml

```diff
@@ -0,0 +1,6 @@
+username:
+  type: str
+
+homedir:
+  type: str
+  default: "/Users/[[ username ]]"
```

Comment:
do we need it? Can't use just ~ ?

Reply:
It depends on `dbt` solely, hopefully it will work with just a `~`. However, here it is just a small example config just for test reasons, so won't it be ok to keep it that way?